content/knowledge-firebase/firebase-crashlytics-dsym-uploading.md

```diff
@@ -83,41 +90,41 @@ If necessary, you can use remote access to the build machine to find the correct
 For Native iOS apps, in the case of using SwiftPackageManager (SPM) instead of CocoaPods, the following script needs to be added in a post-publishing script:
 
 {{< highlight yaml "style=paraiso-dark">}}
-publishing:
-  scripts:
-    - name: Upload debug symbols to Firebase Crashlytics
-      script: | 
-        echo "Find build artifacts"
-        dsymPath=$(find build/ios/xcarchive/* | head -1)
-        echo "dsyms expected in:"
-        ls -d -- $dsymPath/dSYMs/*
-        dsymFile=$(find $dsymPath/dSYMs -name "*.dSYM" | head -1) 
-        if [[ -z ${dsymFile} ]]
-          then
-            echo "No debug symbols were found, skip publishing to Firebase Crashlytics"
-          else
-            echo "Publishing debug symbols in $dsymFile to Firebase Crashlytics"
-            echo $dsymFile
-            ls -d -- $CM_BUILD_DIR/*
-            $HOME/Library/Developer/Xcode/DerivedData/**/SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/upload-symbols \
-              -gsp $CM_BUILD_DIR/<PATH_TO_YOUR_GoogleService-Info.plist> -p ios $dsymFile
-        fi
+  publishing:
+    scripts:
+      - name: Upload debug symbols to Firebase Crashlytics
+        script: | 
+          echo "Find build artifacts"
+          dsymPath=$(find build/ios/xcarchive/* | head -1)
+          echo "dsyms expected in:"
+          ls -d -- $dsymPath/dSYMs/*
+          dsymFile=$(find $dsymPath/dSYMs -name "*.dSYM" | head -1) 
+          if [[ -z ${dsymFile} ]]
+            then
+              echo "No debug symbols were found, skip publishing to Firebase Crashlytics"
+            else
+              echo "Publishing debug symbols in $dsymFile to Firebase Crashlytics"
+              echo $dsymFile
+              ls -d -- $CM_BUILD_DIR/*
+              $HOME/Library/Developer/Xcode/DerivedData/**/SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/upload-symbols \
+                -gsp $CM_BUILD_DIR/<PATH_TO_YOUR_GoogleService-Info.plist> -p ios $dsymFile
+          fi
 {{< /highlight >}}
 
 ### How to upload dSYM artifacts to Firebase Crashlytics using Workflow Editor
 
 In order to upload the dSYM files to Firebase Crashlytics, add the following script to  your **post-publish** script in the Flutter workflow editor:
 
 {{< highlight yaml "style=paraiso-dark">}}
-  echo "Find build artifacts"
-  dsymPath=$(find $CM_BUILD_DIR/build/ios/archive/Runner.xcarchive -name "*.dSYM" | head -1)
-  if [[ -z ${dsymPath} ]]
-  then
-    echo "No debug symbols were found, skip publishing to Firebase Crashlytics"
+  echo "Locating dSYM artifacts..."
+  dsymPath=$(find $CM_BUILD_DIR/build/ios/archive/Runner.xcarchive/dSYMs -name "Runner.app.dSYM" | head -1)
```

Comment:
would be good to add a note to say Runner needs to be updated if they manually changed their project which is by default Runner